code/modules/vehicles/scooter.dm

```diff
@@ -57,6 +57,9 @@
 		H.throw_at(throw_target, 4, 3)
 		H.Knockdown(100)
 		H.adjustStaminaLoss(40)
+		if(!istype(H.head,/obj/item/clothing/head/helmet/) && !istype(H.head,/obj/item/clothing/head/hardhat))
```

Comment:
Instead of checking for these types maybe check for melee armor on any helmet type?

Reply:
we really just want -normal- helmets and hardhats, kind of like a bike helmet. if I did that, literally would have people wearing cake hats to stop brain damage even though they're not strong enough

Reply:
Well the cake hat in specific has no armor so it wouldn't but I know what you mean.

I made myself a list of all the hats and their armor; turns out there's a few weird cases whichever sort of check you go with so just roll with this it's close enough to fine.